code/modules/antagonists/mind_flayer/mindflayer_gamemode.dm

```diff
@@ -0,0 +1,106 @@
+// FOR THE LOVE OF GOD PLACE THIS FILE IN THE RIGHT PLACE WHEN I'M NOT LAZY ENOUGH TO PROPERLY DO IT
+// Yes this is copypasted from Vampire gamemode
+
+/datum/game_mode
+	/// The list of all the mindflayers
+	var/list/datum/mind/mindflayers = list()
```

Comment:
This should probably go on the main game_mode datum

Reply:
Bestie you wrote this code, I have no idea what's going on there